### DISCUSSION TITLE
Running more then one Owncast

### DISCUSSION THREAD

**Discussion by SpiderChat** [owncast/owncast#1385] [Q&A]
Hello
Can running more then one Owncast on same VPS but different user accounts?

**Comment by gabek** [comment]
You can run as many copies as you like, provided they are all listening on different ports.

You can run Owncast with custom ports using command line flags.  Example:

`./owncast --webserverport 8081 --rtmpport 1936`

**Reply by SpiderChat** [reply]
Wow. It's a good news.
Thanks
